frontend/public/components/factory/table.tsx

```diff
@@ -359,7 +359,7 @@ const TableWrapper: React.SFC<TableWrapperProps> = ({
   ...props
 }) => {
   return virtualize ? (
-    <div {...props} role="grid" aria-label={ariaLabel} aria-rowcount={ariaRowCount} />
+    <div {...props} data-test={ariaLabel} role="grid" aria-label={ariaLabel} aria-rowcount={ariaRowCount} />
```

Comment:
pass `data-test` from props and not required here.

Reply:
I found another attribute named "data-test-rows" already found. I am using it. Thank you